HotorNot/app/src/main/res/navigation/nav_graph.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/nav_graph"
+    app:startDestination="@id/mainScreenFragment">
+
+    <fragment
+        android:id="@+id/mainScreenFragment"
+        android:name="com.example.hotornot.fragments.SplashScreenFragment"
+        android:label="MainScreenFragment" >
+        <action
+            android:id="@+id/action_mainScreenFragment_to_mainScreenFragment2"
```

Comment:
Why this action is action_mainScreenFragment_to_mainScreenFragment2.I think the 2nd in the end is unnecessary.If you clear all of them,clean project and create it again you wouldn't have this problem.